README.md

```diff
@@ -606,6 +606,19 @@ If you are updating from **6.14.0 or 6.14.1 on Windows**, use the following step
 
 For more details, see [Critical Bug in Uninstaller for Datadog Agent 6.14.0 and 6.14.1 on Windows][11].
 
+### Ubuntu 20.04 broken by service_facts
+
+Running the `service_facts` module on Ubuntu 20.04 causes the following error:
+
+```
+localhost | FAILED! => {
+    "changed": false,
+    "msg": "Malformed output discovered from systemd list-unit-files: accounts-daemon.service                    enabled         enabled      "
+}
+```
+
+This can be fixed by [updating Ansible to `v2.9.8` or above](https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst#id61).
```

Comment:
```suggestion
To fix this, [update Ansible to `v2.9.8` or above][16].
```

Reply:
and then add at the bottom of the page:
`[16]: https://github.com/ansible/ansible/blob/stable-2.9/changelogs/CHANGELOG-v2.9.rst#id61`

Reply:
You're right ! Done in [4dd2db0](https://github.com/DataDog/ansible-datadog/pull/449/commits/4dd2db0fb53b8ac973732a80986d400e1d4478f3)